src/components/Typeahead.js

```diff
@@ -69,15 +129,65 @@ function Typeahead({ fetchItems, items, title, loading, clearSearch }) {
 
     return (
         <Fragment>
-            <Title text={title} />
-            <SearchInputField
-                placeholder="Search by id or description"
-                onChange={handleSearchTermChange}
-                type="search"
-                variant="outlined"
-                value={searchTerm}
+            {!modal ? <Title text={title} /> : <Fragment />}
+            <InputField
+                adornment={
+                    <svg
+                        xmlns="http://www.w3.org/2000/svg"
+                        width="24"
+                        height="24"
+                        viewBox="0 0 24 24"
+                    >
+                        <path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z" />
```

Comment:
If this is the same as in the `SearchInputField` the path could be in its own component and reused?